dev/tasks/linux-packages/apache-arrow/debian/rules

```diff
@@ -88,19 +89,9 @@ override_dh_auto_install:
 	  --sourcedirectory=c_glib		\
 	  --builddirectory=c_glib_build		\
 	  --buildsystem=meson+ninja
-	# Remove built files to reduce disk usage
-	dh_auto_clean				\
```

Comment:
just a question, is this removal of `dh_auto_clean` required because it was removing some required debug symbols or something like this? or is just unrelated to the other change?

Reply:
The latter. Sorry for mixing unrelated changes...

Reply:
no worries at all! I was just wondering if it had anything to do that I might be missing!